### DISCUSSION TITLE
zotero7.0.0 beta.63安装了0.6.3但是没有起作用，和没有安装的时候显示的东西一样，请问是兼容性问题吗？还是我需要一些额外的设置？

### DISCUSSION THREAD

**Discussion by YanChuyue** [MuiseDestiny/zotero-reference#239] [Q&A]
如题

**Comment by Leo9209** [comment]
我也是安装了zotero reference0.6.3,但右侧边栏没有找到参考文献按钮，翻译按钮也没有。
![2](https://github.com/MuiseDestiny/zotero-reference/assets/160202711/c14f7921-1797-4872-8312-b5d570e5baf3)
![无标题](https://github.com/MuiseDestiny/zotero-reference/assets/160202711/34f2ab5c-8948-4f1d-9e33-f4ba142a2a31)
